### PR TITLE
magento/devdocs#:  Develop issue 1740. Additional changes

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) provides additional fixes for https://github.com/magento/devdocs/pull/5298/

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/config-guide/cli/config-cli-subcommands-cron.html
- https://devdocs.magento.com/guides/v2.3/config-guide/cli/config-cli-subcommands-cron.html

## Links to Magento source code

- ...

Thank you!

CC: @jeff-matthews @hostep

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: _includes/config/setup-cron_2.2_how-to.md [hunk 1]
```diff
@@ -29,17 +29,17 @@ crontab -l
 A sample follows:
 
 ```terminal
-#~ MAGENTO START
-* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run | grep -v Ran jobs by schedule >> /var/www/html/magento2/var/log/magento.cron.log
-* * * * * /usr/bin/php /var/www/html/magento2/update/cron.php >> /var/www/html/magento2/var/log/update.cron.log
-* * * * * /usr/bin/php /var/www/html/magento2/bin/magento setup:cron:run >> /var/www/html/magento2/var/log/setup.cron.log
-#~ MAGENTO END
+#~ MAGENTO START c5f9e5ed71cceaabc4d4fd9b3e827a2b
+#* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run 2>&1 | grep -v "Ran jobs by schedule" >> /var/www/html/magento2/var/log/magento.cron.log
```
Comment (rogyar) [inline _includes/config/setup-cron_2.2_how-to.md]: All crontab records are commented by `#` for some reason. I believe we should not do that if we are showing the working example.

Comment (atwixfirster) [reply _includes/config/setup-cron_2.2_how-to.md]: Good catch, @rogyar 

Thank you!

File: _includes/config/setup-cron_2.2_how-to.md [hunk 2]
```diff
@@ -29,17 +29,17 @@ crontab -l
...
 * * * * * /usr/bin/php /var/www/html/magento2/bin/magento setup:cron:run >> /var/www/html/magento2/var/log/setup.cron.log
-#~ MAGENTO END
+#~ MAGENTO END c5f9e5ed71cceaabc4d4fd9b3e827a2b
 ```
 
 {%
 include note.html
 type='info'
-content='The `update/cron.php` file exists in [Composer](https://glossary.magento.com/composer)- and [archive-based](https://magento.com/tech-resources/download) Magento installations. It does not exist if you installed Magento by cloning a git repository.
+content='The `update/cron.php` file exists in [Composer](https://glossary.magento.com/composer)- and [archive-based](https://magento.com/tech-resources/download) Magento installations. It does not exist if you installed Magento by cloning a [Magento 2 git repository](https://github.com/magento/magento2/).
```
Comment (hostep) [inline _includes/config/setup-cron_2.2_how-to.md]: Please change "if you installed Magento by cloning **a** Magento 2 git repository" to "if you installed Magento by cloning **the** Magento 2 git repository".

Same remark for the 2.3 file.

Comment (atwixfirster) [reply _includes/config/setup-cron_2.2_how-to.md]: @hostep , 

**done**

Thank you!

**Comment by dobooth** [comment]
running tests

**Pull request merged**